### ISSUE TITLE
Automate generation of dot-webcomponents versions

### ISSUE THREAD

**Issue by nollymar** [dotCMS/core#20245]
Currently, when a change in dot-webcomponent is made, once the PR is merged into master (or release branch) a developer needs to generate and publish the version manually. The process is summarized as follows: 

1.  From core-web run `npm run nx build dotcms-webcomponents --with-deps`
2. Set the proper version in the file  `core-web/dist/libs/dotcms-webcomponents/package.json` 
3. Publish the component using `npm publish`
4. Update the webcomponents version in core project only when the PR is merged into a **release branch**, editing the file https://github.com/dotCMS/core/blob/master/dotCMS/gradle.properties#L3 with the version published on **step 3**

The idea is to automate the process when a PR (that **ONLY** affects libs/dotcms-webcomponents) gets merged , the same way we do it for core-web via GitHub Actions.

**Comment by nollymar** [comment]
PRs: 
https://github.com/dotCMS/dev-scripts/pull/3
https://github.com/dotCMS/core-web/pull/1668
https://github.com/dotCMS/core-web/pull/1665
https://github.com/dotCMS/core/pull/20561
